sdk/identity/identity/test/manual-integration/AzureFunctions/IdentityTest/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es6",
+    "outDir": "dist",
+    "rootDir": ".",
+    "sourceMap": true,
+    "strict": false
```

Comment:
Should this be false?

Reply:
Hmm, I don't think it matters for now.

Reply:
(yeah, this was another "customer facing" thing, although our example code should work with strict!)